src/lib/tabs/tabs-animations.ts

```diff
@@ -22,8 +22,14 @@ export const matTabsAnimations: {
   translateTab: trigger('translateTab', [
     // Note: transitions to `none` instead of 0, because some browsers might blur the content.
     state('center, void, left-origin-center, right-origin-center', style({transform: 'none'})),
-    state('left', style({transform: 'translate3d(-100%, 0, 0)'})),
-    state('right', style({transform: 'translate3d(100%, 0, 0)'})),
+
+    // If the tab is either on the left or right, we additionally add a `min-height` of 1px
+    // in order to ensure that the element has a height before its state changes. This is
+    // necessary because Chrome does seem to skip the transition in RTL mode if the element does
+    // not have a static height and is not rendered. See related issue: #9465
+    state('left', style({transform: 'translate3d(-100%, 0, 0)', minHeight: '1px'})),
```

Comment:
Can we do this directly in the stylesheet? I think we have a class for the off-screen tabs which should allow us to set the `min-height`.

Reply:
We only have a class for the `active` tab body, so we could work with `:not`, but since this technically belongs to the animation setup I'd anyway prefer having this here.

That way if someone looks for the animations, he can quickly see that there is some special workaround applied.

Reply:
True, but this makes it a lot more difficult for somebody to change the `min-height`, if they wanted to.

Reply:
That `min-height` is only temporarily while the tab is hidden. As soon as it transitions to the other state, the `min-height` is completely gone.

Reply:
I see your point with the low-effort to place in the stylesheet. In regards to the code size, I think the generated CSS will take up more space than how it's right now. 

Also, if we consider it like that, we could also argue about moving the actual `transform`'s that don't include any computed values to the stylesheet.

Anyway, I will switch it to the CSS but keep a comment that mentions the Chrome workaround that is placed in the stylesheet.

Reply:
As discussed on Slack; since depending on the `.mat-tab-body-active` does not work because the class is being added before hidden tabs start the actual animation, we are keeping the workaround in the TS file.